.github/CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 
 # Services
 /core/services/directrequest @smartcontractkit/foundations
-/core/services/feeds @smartcontractkit/op-core @eutopian @yevshev
+/core/services/feeds @smartcontractkit/deployment-automation @eutopian @yevshev
```

Comment:
FYI - we try to avoid individual owners where we can and prefer use of sub-teams/teams